### DISCUSSION TITLE
Add scalar JSON to gqlgen

### DISCUSSION THREAD

**Discussion by Selen34** [99designs/gqlgen#1882] [Ideas]
Please add scalar JSON to supported types. JSON is just map[string]interface{}.
You added Map. But nobody called JSON "Map". All who uses GraphQL called json - JSON.

**Comment by frederikhors** [comment]
@StevenACoffman do you have any news on how to deal with these "feature requests"? Did you have the meeting? Do I move everything into discussions/ideas?

**Comment by Selen34** [comment]
I think it's need to move to discussions. More and more services uses apollo federation. But most of them written in Node.js and they uses "JSON scalar".

**Comment by StevenACoffman** [comment]
@frederikhors So we talked and it seems like it's more of a judgment call, if that's ok. 

When it's an actionable, concrete proposal that makes a case for why it is valuable, demonstrating that it is reasonably well researched, then it belongs as a feature request issue for easy tracking by others and downstream projects.

When someone more casually mentions something, that should instead be a discussion that should first get better fleshed out by the rest of the community as to what and why should be added.

Is that fair?

**Reply by frederikhors** [reply]
Great, I agree 100%.

**Comment by akbari4yaseen** [comment]
I have a column in Postgres with the data type of JSON, when I get the data, I lose the format and get messy plain text. Is there anyone to help?
